rfs/src/fungi/meta.rs

```diff
@@ -268,6 +268,14 @@ impl Reader {
         Ok(results)
     }
 
+    pub async fn all_blocks(&self) -> Result<Vec<Block>> {
```

Comment:
This can be A LOT for big flists. that has a lot of big files. You need to do this with pages. It can be implemented as an iterator as well or async stream